pkg/cmd/cli/backup/create.go

```diff
@@ -281,6 +284,26 @@ func (o *CreateOptions) Run(c *cobra.Command, f client.Factory) error {
 	return nil
 }
 
+// parseOrderedResources converts to map of Kinds to an ordered list of specific resources of that Kind.
+// Resource names in the list are in format 'namespace/resourcename' and separated by commas.
+// Key-value pairs in the mapping are separated by semi-colon.
+// Ex: 'pods=ns1/pod1,ns1/pod2;persistentvolumeclaims=ns1/pvc4,ns1/pvc8'.
+func parseOrderedResources(orderStr string) (map[string]string, error) {
+	orders := strings.Split(orderStr, ";")
```

Comment:
I want to suggest better names for `orders`, `orderMap`, and `attrs`. But I am failing to come up with better names. I'll keep thinking.

Reply:
orderStr ==> orderMapStr
orders ==> entryStrs
orderMap ==> orderedResources
attrs ==> kv